spec/support/set_current_pack_helper.rb

```diff
@@ -0,0 +1,9 @@
+def set_current_pack(email, password, password_confirmation, current_pack_id)
```

Comment:
Unused method argument - `current_pack_id`. If it's necessary, use `_` or `_current_pack_id` as an argument name to indicate that it won't be used.